private_gpt/ui/ui.py

```diff
@@ -409,11 +410,30 @@ def _build_ui_blocks(self) -> gr.Blocks:
                         inputs=system_prompt_input,
                     )
 
+                def get_model_label() -> str | None:
+                    # Determine the model label based on PGPT_PROFILES env variable.
+                    pgpt_profiles = os.environ.get("PGPT_PROFILES")
```

Comment:
Do not rely on PGPT_PROFILES, given the potential profiles are up to the user and not standardized. 
Better use settings().llm.mode. Possible options: "local", "openai", "openailike", "sagemaker", "mock", "ollama".